.changeset/angry-hornets-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lg-chat/message': minor
```

Comment:
Can you add a changeset for rich links?

Reply:
Sure added one. I went with "patch" but wasn't sure given this is an initial release.